pom.xml

```diff
@@ -290,6 +290,7 @@
                 <version>3.4.1</version>
                 <configuration>
                     <source>${maven.compiler.source}</source>
+                    <doclint>all,-syntax</doclint>
```

Comment:
Is this the same issue as with the SDK where `javadoc` picks up an annotation as a doctag?

Reply:
yes.  and i can't figure out why or reproduce it 😞